src/datasources/human-description-api/entities/human-description.entity.ts

```diff
@@ -0,0 +1,54 @@
+export enum ValueType {
+  Word = 'word',
+  TokenValue = 'tokenValue',
+  Identifier = 'identifier',
+  Address = 'address',
+  Decimals = 'decimals',
+}
+
+type TokenValueType = {
+  amount: bigint;
+  address: string;
+};
+
+interface TokenValueFragment {
+  type: ValueType.TokenValue;
+  value: TokenValueType;
+}
+
+interface WordFragment {
+  type: ValueType.Word;
+  value: string;
+}
+
+interface IdentifierFragment {
+  type: ValueType.Identifier;
+  value: unknown;
+}
+
+interface AddressFragment {
+  type: ValueType.Address;
+  value: string;
+}
+
+interface DecimalsFragment {
+  type: ValueType.Decimals;
+  value: unknown;
+}
+
+export type HumanReadableFragment =
+  | WordFragment
+  | TokenValueFragment
+  | IdentifierFragment
+  | AddressFragment
+  | DecimalsFragment;
+
+type MessageTemplate = {
+  process: (to: string, params: readonly unknown[]) => HumanReadableFragment[];
+};
+
+export type MessageTemplates = Record<string, MessageTemplate>;
+
+export type Expression = {
+  [key: string]: string;
+};
```

Comment:
What is the value of this type? We can use `Record<string, string>` directly to accomplish the same constraints that we have for reading JSON.

Reply:
I like the `Record` syntax so I've changed it. It should be the same as before just so we don't have `any` when parsing the json.